docs/developer-guides/native-token-reserve-management.md

```diff
@@ -0,0 +1,441 @@
+
+- `reserve-handover` &ndash; a command for finishing the flow (either there is nothing to release or the Governance Authority user wants to finish the flow).
+- `reserve-release-funds` &ndash; a command to "move" the flow until the end. 
+
+## 2. Releasing tokens from reserve to circulation
+
+Objective: Release available reserve tokens (defined by the `V` function).
+
+### 2.1 Run the `reserve-release-funds` command
+
+#### Command template
+
+```bash
+./pc-contracts-cli reserve-release-funds \
+--genesis-utxo <GenesisCommitteeHashUTxO> \
+--ogmios-host localhost \
+--ogmios-port 1337 \
+--kupo-host localhost \
+--kupo-port 1442 \
+--payment-signing-key-file <PaymentSigningKeyFile> \
+--network <Network> \
+--reserve-transaction-input 156803a18a4dbde48dc83a21b8b1007b693cdf096b73596252745fdef1de6414#0 \
+--total-accrued-till-now 5
+```
+
+#### Explanation of parameters
+
+* `--genesis-utxo`: The genesis UTXO of the running partnerchain
+
+* `--network`: The network you operate on (for example, `testnet` or `mainnet`)
+
+* `--payment-signing-key-file`: Your payment signing key file for transaction signing
+
+* `--reserve-transaction-input`: UTXO where the `V` function script is attached as a reference
+
+* `--total-accrued-till-now`: amount of tokens to release 
+
+   - The number should be calculated as `number of tokens release up to this time + amount of tokens to release`
+
+   - If 10 tokens have already been released and you want to release 10 more, then the total accrued until now should be defined as 20. 
+
+   - Ensure you have enough tokens to release in this time frame (the `V` function defines release logic). 
+
+#### Steps
+
+1. Execute the `reserve-release-funds` command.
+
+2. Confirm completion: make sure that the release process has been completed by checking the command output.
+
+## 2.2 Transferring token control to smart contracts
+
+Objective: Complete the reserve setup by handing over control to the appropriate scripts.
+
+### 2.2.1 Run the `reserve-handover` command
+
+#### Command template
+
+```bash
+./pc-contracts-cli reserve-handover \
+  --genesis-utxo <GenesisCommitteeHashUTxO> \
+  --ogmios-host <Ogmios_Host> \
+  --ogmios-port <Ogmios_Port> \
+  --kupo-host <Kupo_Host> \
+  --kupo-port <Kupo_Port> \
+  --payment-signing-key-file <PaymentSigningKeyFile> \
+  --network <Network>
+  --hand-over
+```
+
+#### Steps
+
+1. Execute the `reserve-handover` command.
+
+2. Confirm completion: Make sure that the handover process has been completed and the illiquid circulation supply transaction has been confirmed by checking the command output.
+
+## 2.3 Initializing token reserve with initial deposit
+
+Objective: Configure the initial token reserve by depositing a specified amount of tokens into the reserve contract.
+
+
+# Part Three: Configuration
+
+Configuration includes `reserve-deposit` and `reserve-update-settings` commands used to either change token in reserve or V-function. 
```

Comment:
"reserve deposit" is usage. It is not configuration. Only "initialize", "create" and "update settings" are related to configuration.

"release" and "deposit" are for sure usage.

Reply:
Then let's have two sections - configuration and usage.

Reply:
In that case, do we still need a separate section for "Initialization"? 

I've moved the `reserve-deposit` section to the end of the "Part Two: Usage" section.